### DISCUSSION TITLE
Accounts imported in bulk via CSV are encountering certificate parameter issues during login

### DISCUSSION THREAD

**Discussion by de7ang** [rejetto/hfs#576] [Q&A]
Accounts imported in bulk via CSV are encountering certificate parameter issues during login, though the passwords are entered correctly.
![微信截图_20240429162949](https://github.com/rejetto/hfs/assets/108105714/2792c2b7-bc31-4132-8ca0-7d82252515f0)

**Comment by rejetto** [comment]
i would need the english error to dig more, but i see that it's about the certificate.
I don't think this is a bug in hfs, but some problem with your certificate.
I'm quite sure it has nothing to do with csv and accounts.
Try creating an account manally, and you'll see the same error.

**Comment by de7ang** [comment]
When I import an account using CSV (as shown in the picture), it prompts me with certificate parameter issues. However, when I directly go to that account and change the password, I can log in normally with the new password. Alternatively, if I manually create an account, I can also log in normally. Therefore, I suspect whether there is a hash error in the accounts imported via CSV. I observed that after changing the password, the SRP value under the accounts field in config.yaml changes. Is there a discrepancy between the encryption algorithm used for CSV import and the one used for manual creation?
![导入](https://github.com/rejetto/hfs/assets/108105714/070cc3fa-17e0-4a86-8476-08c8adc66c28)
